teorija.tex

```diff
@@ -4534,10 +4540,10 @@ \subsection{Bisekcija}
   \begin{algorithmic}[1]
     \While{$|b-a| < \epsilon$}
         \State $s \gets \frac{a+b}{2}$
-        \If{\textsc{predznak}$(f(a)) =\ $\textsc{predznak}$(f(b))$}
-          \State $a \gets s$
-        \Else
+        \If{\textsc{predznak}$(f(a)) =\ $\textsc{predznak}$(f(s))$}
```

Comment:
Ja to pa je moja napaka, sem mislil isto kot ti ampak sem potem narobe popravil.